### PR TITLE
Create pause menu

### LINKED ISSUE #445: Add support for skipping to the next or previous chapter in a video
**Describe the feature you'd like**

I have video files which have chapters, but I can't see an OSD button or key combination on the Roku remote for skipping from chapter to chapter. If I hover over the timeline in the web player, I can see the chapter names, so I know Jellyfin is reading them, but there doesn't appear to be a way to navigate between them in the Roku app. One idea would be to make a long-press of the forward/back buttons on the Roku remote trigger skipping to the next/previous chapter, while a short press of those buttons has the existing functionality of fast-forwarding/rewinding.

Adding this functionality would greatly improve the user experience of the Roku app with videos that have chapters.

**Additional context**

Thank you for considering this feature request!

### PR DESCRIPTION
**Pull request by 1hitsong**
## Changes
Creates a menu displayed when the user pauses playback of a video. **The trickplay bar is no longer shown.** This changes current behavior. The trickplay bar is still accessed by pressing left, right, rewind, or fast forward on the remote while the video is playing.

The menu has 3 buttons: 
1. List of all found chapters with current chapter in focus
2. Previous chapter
3. Next Chapter

The menu is designed to handle more icons and functions in the future.

If user tries to go before chapter 1, it simply replays the chapter from the start. User can not go beyond last chapter, instruction is ignored.

Chapter names are used in the list, so if user has a chapter named "It's All Over" instead of "Chapter 1," "It's All Over" will be displayed.

If no chapter data is found, buttons are still displayed, but are inactive. Chapter list displays "No chapter data found" and is also inactive.

## Issues
Fixes #445

## Notes
This PR uses a param introduced in Roku OS 10, video.seekMode. This param is needed to make the chapter selection accurate.

Determining current chapter isn't perfect. We must compare Roku's video position param(seconds) to the chapter start time (ticks), so we must convert one of the numbers to the other format. The converted number drifts away from accuracy, especially late in movies when the values are at their highest. This can cause the current chapter lookup to be incorrect. As a result, late in a video a user may find they need to wait 2-3 seconds for the video position to advance past the chapter start value so they can skip to the next chapter. This issue can also cause the chapter list auto focus to be 1 chapter behind late in a movie.

**Update**. Adding a 15 second buffer to the current position fixed the issue I mentioned above. This fix is reflected in this PR.

## Screenshot
pauseMenu

## Demo
https://social.linux.pizza/@tgpo/111327355574987355

### CODE REVIEW

**Comment by lakerssuperman** [comment]
Wow.  This looks awesome!  Special points for include a A Nightmare on Elm Street Part 2 screenshot!

File: components/video/PauseMenu.brs [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import "pkg:/source/utils/misc.brs"
+
```
Comment (cewert) [inline components/video/PauseMenu.brs]: New files should be `.bs`

**Review by cewert — CHANGES_REQUESTED** [review]

**Comment by cewert** [comment]
The code looks good.

Right now the next chapter buttons are called `previous` and `next` but maybe we should rename them to `previous-chapter` and `next-chapter`? Eventually we will need icons for previous item(in playlist) and next item too.

File: components/Clock.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component name="Clock" extends="Group" initialFocus="chapterNext">
```
Comment (cewert) [inline components/Clock.xml]:
```suggestion
<component name="Clock" extends="Group">
```

This looks like a leftover from a copy/paste

**Review by cewert — CHANGES_REQUESTED** [review]

**Review by cewert — APPROVED** [review]
Works great. Nice job on this 👍

**Pull request merged**